android-lib/src/main/java/com/kinvey/android/cache/ClassHash.java

```diff
@@ -217,12 +217,24 @@ public static DynamicRealmObject saveData(String name, DynamicRealm realm, Class
         if (obj.containsKey("_kmd")){
             Map kmd = (Map)obj.get("_kmd");
             if (kmd != null) {
-                kmd.put("lmt", String.format("%tFT%<tTZ",
-                        Calendar.getInstance(TimeZone.getTimeZone("Z"))));
+                KinveyMetaData metadata = new KinveyMetaData();
```

Comment:
A cleaner way to do this is to expose a static method on the `KinveyMetaData` class that constructs the class from a `Map`. So you can call something like - 
`KinveyMetaData.buildFromMap(kmd)`.

This encapsulates the keys and any other logic for dates etc. within the `KinveyMetadata` class.